Test failure: Error resolving real path of 'spec/std/data/test_file.txt'

Hello! I cloned the crystal repo (up to commit 54e68f0b3f5399c4574a673b55fd830850cf9193), and ran all of the tests with `make spec`. I got one test failure:

```
Using /usr/local/opt/llvm@8/bin/llvm-config [version=8.0.1]
.build/all_spec
...
Failures:

  1) File symlink creates a symbolic link

       Error resolving real path of 'spec/std/data/test_file.txt': No such file or directory (Errno)
         from src/crystal/system/unix/file.cr:92:5 in 'real_path'
         from src/file.cr:559:5 in 'real_path'
         from spec/std/file_spec.cr:247:22 in '->'
         from src/spec/example.cr:255:3 in 'internal_run'
         from src/spec/example.cr:35:16 in 'run'
         from src/spec/context.cr:296:23 in 'internal_run'
         from src/spec/context.cr:291:7 in 'run'
         from src/spec/context.cr:296:23 in 'internal_run'
         from src/spec/context.cr:291:7 in 'run'
         from src/spec/context.cr:48:23 in 'run'
         from src/spec/dsl.cr:270:7 in '->'
         from src/kernel.cr:255:3 in 'run'
         from src/crystal/main.cr:47:5 in 'main'
         from src/crystal/main.cr:106:3 in 'main'


Finished in 38:47 minutes
18312 examples, 0 failures, 1 errors, 17 pending

Failed examples:

crystal spec spec/std/file_spec.cr:244 # File symlink creates a symbolic link
make: *** [spec] Error 1
```

* macOS 10.15.1

```
$ .build/crystal --version
Crystal 0.32.0-dev [54e68f0b3] (2019-12-02)

LLVM: 8.0.1
Default target: x86_64-apple-macosx
```

--- a/src/lib_c/x86_64-darwin/c/stdlib.cr
+++ b/src/lib_c/x86_64-darwin/c/stdlib.cr
@@ -17,7 +17,7 @@ lib LibC
   fun mkstemps(x0 : Char*, x1 : Int) : Int
   fun putenv(x0 : Char*) : Int
   fun realloc(x0 : Void*, x1 : SizeT) : Void*
-  fun realpath(x0 : Char*, x1 : Char*) : Char*
+  fun realpath = "realpath$DARWIN_EXTSN"(x0 : Char*, x1 : Char*) : Char*
   fun setenv(x0 : Char*, x1 : Char*, x2 : Int) : Int
   fun strtof(x0 : Char*, x1 : Char**) : Float
   fun strtod(x0 : Char*, x1 : Char**) : Double

